src/openforms/js/lang/nl.json

```diff
@@ -271,6 +271,11 @@
     "isTranslated": true,
     "originalDefault": "minus"
   },
+  "5zEEiB": {
+    "defaultMessage": "Om het Datatype te veranderen, moet ook de beginwaarde aangepast worden. In sommige gevallen kan dit betekenen dat de beginwaarde wordt veranderd naar de standaard waarde van het Datatype. Weet je zeker dat je dit wilt doen?",
```

Comment:
this is a bit too technical I think, the english version was simplified a bit.

Also please lower case "datatype"